janMarkusMilan/assignments/twopointers/lc977/lc977.js

```diff
@@ -1,12 +1,56 @@
 // Problem Statement #
 // Given a sorted array, create a new array containing squares of all the numbers of the input array in the sorted order.
 
+/*
+First Thoughts (Brute Force):
+  - Loop through the array
+  - Square the input after each iteration (if we create a new array, then just insert the new values into the new array)
+  - Sort from least to greatest
+
+Second Thoughts (Two Pointer):
+  - Make an array and fill with some arbitrary number
+  - Define two pointers at beginning and end
+  - Get the square at each value the pointers are on
+  - On some condition, we pick the most suitable square and insert into the array
+*/
+
```

Comment:
I love that you thought-through and coded both solutions! For your own personal thinking (no need to answer), what is the time and space complexity of each?